all/099-generic.yml

```diff
@@ -59,6 +59,7 @@ required_packages_default:
   - pv
   - python-is-python3
   - rsyslog
+  - "{{ 'rng-tools-debian' if ansible_distribution == 'ubuntu' and ansible_distribution_version == '22.04'  else 'rng-tools' }}"
```

Comment:
I'm not really sure what to think about this.

a) this conditional construct looks weird in this context to me. maybe this is an indication that this is not the right location to set this
b) do we really want to install this on any node? the commit message indicates only compute nodes to make use of it